projects/vdk-plugins/airflow-provider-vdk/vdk_provider/hooks/vdk.py

```diff
@@ -37,35 +37,27 @@ def start_job_execution(self) -> None:
         API path: POST /data-jobs/for-team/{team_name}/jobs/{job_name}/deployments/{deployment_id}/executions
 
         :return: None
-
+        """
         self.method = "POST"
 
         endpoint = f"/data-jobs/for-team/{self.team_name}/jobs/{self.job_name}/deployments/{self.deployment_id}/executions"
```

Comment:
Can we extract this endpoint from vdk-control-service-api client?

Reply:
I spent a lot of time trying to get it to work and I can't seem to get the auth token to pass through to the API call. I'd rather not spend 2 days debugging this so if it's okay with you I'd rather keep it hardcoded for now given we aren't planning any API path changes any time soon (We aren't, correct?). We can create a story for amending the provider to use vdk-control-service-api.

Reply:
Let's see it together first. It seems a small thing but I don't like keeping technical debt if we can avoid it. It's a death by a thousand cuts .